grpc/interceptors.go

```diff
@@ -7,20 +7,38 @@ import (
 	"google.golang.org/grpc"
 )
 
-// defaultMethodTimeout is the default context timeout for all inbound gRPC
+// DefaultMethodTimeout is the default context timeout for all inbound gRPC
```

Comment:
`DefaultMethodTimeout` now governs both inbound SDK->RDK/RDK->module gRPC methods and outbound RDK->module methods. Can we update the comment here? Separately, would it be wise to have `DefaultInboundMethodTimeout` and `DefaultOutboundMethodTimeout`? I think the answer is no, but curious what others think.

Reply:
Is that intended? I thought we only wanted to add this timeout client-side.

If the answer is "yes" then yeah we should document appropriately.

Reply:
I would also think no since this is a timeout on default contexts and I don't see a reason for differentiating between outbound and inbound requests. If they need different timeouts for some reason, it is probably best to explicitly define that.